open-metadata-implementation/adapters/open-connectors/repository-services-connectors/open-metadata-collection-store-connectors/graph-repository-connector/pom.xml

```diff
@@ -82,6 +124,12 @@
                                 <!-- Runtime dependencies for janusgraph-->
                                 <ignoredUnusedDeclaredDependency>org.janusgraph:janusgraph-berkeleyje:*</ignoredUnusedDeclaredDependency>
                                 <ignoredUnusedDeclaredDependency>org.janusgraph:janusgraph-lucene:*</ignoredUnusedDeclaredDependency>
+                                <ignoredUnusedDeclaredDependency>org.janusgraph:janusgraph-es:*</ignoredUnusedDeclaredDependency>
```

Comment:
WHy do we have these exclusions. they need to be clarified.

If a component is a runtime only dependency then I understand. I can see you recorded antlr as runtime in the pom - and the artifact is named runtime, so that makes sense -- but why the others?

There could be latent issues here that need investigation

Reply:
All these dependencies are runtime, used by janusgraph.

Reply:
Can you update the pom to add `<scope>runtime</scope>` for these - clarity and stops them being picked up at compile time.

Reply:
* joda time is pre-java 8 - but agree it's being used on our dependencies inc. tinkerpop and janus
Agree with others for now - 

Janus, cassandra , tinkerpop are bringing in rather a log of dependencies. I can see the rationale for all of these changes. When we are able to dynamically load - and also have the option of client/server, we'll be in a much cleaner state for the rest of the code.

For now, let's merge.. and then deal with any issues in later PRs as needed